test/integration/cmn/network.go

```diff
@@ -140,6 +192,39 @@ func RobotPortNames() []nwo.PortName {
 	return []nwo.PortName{nwo.ListenPort}
 }
 
+// ChannelTransferPortNames returns the list of ports that need to be reserved for the Channel Transfer service
+func ChannelTransferPortNames() []nwo.PortName {
+	return []nwo.PortName{nwo.ListenPort, GrpcPort, HttpPort}
+}
+
+// ChannelTransferPort returns the named port reserved for the Channel Transfer instance
+func (n *NetworkFoundation) ChannelTransferPort(portName nwo.PortName) uint16 {
+	ports := n.ChannelTransfer.Ports
+	Expect(ports).NotTo(BeNil())
+	return ports[portName]
```

Comment:
You should probably do a check for a specific port. See how it is done in fabric tests.